common/src/main/java/tachyon/util/io/FileUtils.java

```diff
@@ -152,4 +152,17 @@ public static void createBlockPath(String path) throws IOException {
   public static void move(File from, File to) throws IOException {
     Files.move(from, to);
   }
+
+  /**
+   * Delete the file or directory
```

Comment:
I think either is fine as long as we specify those corner cases clearly in javadoc.